.svgo.yml

```diff
@@ -0,0 +1,5 @@
+plugins:
+  - removeViewBox: false
+  - addAttributesToSVGElement:
+    - attributes:
```

Comment:
Incorrect yaml here. Remove the dash to get rid of the console errors.

Reply:
Weird, http://www.yamllint.com/ listed that as a "correct" format. Dashes are now removed and the console errors are no longer visible